eng/common/templates/steps/init-docker-linux.yml

```diff
@@ -34,16 +34,45 @@ steps:
       -f $(engCommonPath)/Dockerfile.WithRepo .
     displayName: Build Image for Image Builder
     condition: and(succeeded(), ${{ parameters.condition }})
-  - script: >
-      echo "##vso[task.setvariable variable=runImageBuilderCmd]
-      docker run --rm
-      -v /var/run/docker.sock:/var/run/docker.sock
-      -v $(Build.ArtifactStagingDirectory):$(artifactsPath)
-      -w /repo
-      $(imageBuilderDockerRunExtraOptions)
-      $(imageNames.imageBuilder.withrepo)"
-    displayName: Define runImageBuilderCmd Variable
+  - task: PowerShell@2
```

Comment:
Is it better to use a hash commit version than PowerShell@v2 where 2 could represent something different under the hood?  I konwe dependabot supports updating hash commits with a comment e.g. PowerShell@12321qwerrtytughfgf4g4544543 # v2.0.0